test/test-format.ts

```diff
@@ -132,116 +139,119 @@ test.serial(
       },
       async fixturesDir => {
         const okay = await format.format(OPTIONS, [], true);
-        t.is(okay, true);
+        assert.ok(okay);
         const contents = fs.readFileSync(
           path.join(fixturesDir, 'a.ts'),
           'utf8'
         );
-        t.deepEqual(contents, GOOD_CODE);
+        assert.deepStrictEqual(contents, GOOD_CODE);
       }
     );
-  }
-);
+  });
 
-test.serial('skip files listed in exclude', t => {
-  return withFixtures(
-    {
-      'tsconfig.json': JSON.stringify({ exclude: ['b.*'] }),
-      'a.ts': GOOD_CODE,
-      'b.ts': BAD_CODE,
-    },
-    async () => {
-      const okay = await format.format(OPTIONS);
-      t.is(okay, true);
-    }
-  );
-});
+  it('skip files listed in exclude', () => {
+    return withFixtures(
+      {
+        'tsconfig.json': JSON.stringify({ exclude: ['b.*'] }),
+        'a.ts': GOOD_CODE,
+        'b.ts': BAD_CODE,
+      },
+      async () => {
+        const okay = await format.format(OPTIONS);
+        assert.ok(okay);
+      }
+    );
+  });
 
-test.serial('format globs listed in include', t => {
-  return withFixtures(
-    {
-      'tsconfig.json': JSON.stringify({ include: ['dirb/*'] }),
-      dira: { 'a.ts': GOOD_CODE },
-      dirb: { 'b.ts': BAD_CODE },
-    },
-    async () => {
-      const okay = await format.format(OPTIONS);
-      t.is(okay, false);
-    }
-  );
-});
+  it('format globs listed in include', () => {
+    return withFixtures(
+      {
+        'tsconfig.json': JSON.stringify({ include: ['dirb/*'] }),
+        dira: { 'a.ts': GOOD_CODE },
+        dirb: { 'b.ts': BAD_CODE },
+      },
+      async () => {
+        const okay = await format.format(OPTIONS);
+        assert.strictEqual(okay, false);
```

Comment:
Nit -- I see both `assert.strictEqual(okay, false)` and `assert.ok(!okay)`, can we just use one of those?

Reply:
yeah that's gross, good feedback